src/components/application_manager/src/app_launch/app_launch_data_json.cc

```diff
@@ -82,12 +83,14 @@ Json::Value& AppLaunchDataJson::GetApplicationData() const {
 }
 
 Json::Value& AppLaunchDataJson::GetApplicationListAndIndex(
-    const ApplicationData& app_data, int32_t& founded_index) const {
+    const ApplicationData& app_data,
+    int32_t& founded_index,
```

Comment:
@YarikMamykin `founded` -> `found` ?

Reply:
@AByzhynar , fixed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/1a5c4105394dc643ba414dbf70deb52afbc045ea